api/tests/unit/integrations/test_unit_integration.py

```diff
@@ -0,0 +1,106 @@
+from integrations.amplitude.models import AmplitudeConfiguration
+from integrations.integration import identify_integrations
+from integrations.segment.models import SegmentConfiguration
+
+
+def test_identify_integrations_amplitude_called(mocker, environment, identity):
+    # Given
+    mock_amplitude_wrapper = mocker.patch(
+        "integrations.amplitude.amplitude.AmplitudeWrapper.identify_user_async"
+    )
+    AmplitudeConfiguration.objects.create(api_key="abc-123", environment=environment)
+
+    # When
+    identify_integrations(identity, identity.get_all_feature_states())
+
+    # Then
+    mock_amplitude_wrapper.assert_called()
+
+
+def test_identify_integrations_segment_called(mocker, environment, identity):
+    # Given
+    mock_segment_wrapper = mocker.patch(
+        "integrations.segment.segment.SegmentWrapper.identify_user_async"
+    )
+    SegmentConfiguration.objects.create(api_key="abc-123", environment=environment)
+    # When
+    identify_integrations(identity, identity.get_all_feature_states())
+
+    # Then
+    mock_segment_wrapper.assert_called()
+
+
+def test_identify_integrations_segment_and_amplitude_called(
```

Comment:
@matthewelwell I think this test is mostly useless since, `test_identify_integrations_calls_every_integration_in_identity_integrations_dict` covers this part?

Reply:
Yep, agreed.